Riot/Modules/Room/CreationModal/RoomCreationEventsModal/RoomCreationEventsModalViewModel.swift

```diff
@@ -136,9 +127,34 @@ final class RoomCreationEventsModalViewModel: RoomCreationEventsModalViewModelTy
     // MARK: - Private
     
     private func loadData() {
+        events.removeAll()
+        
+        //  shape-up events
+        for event in roomState.stateEvents {
+            let formatterError = UnsafeMutablePointer<MXKEventFormatterError>.allocate(capacity: 1)
+            if !shouldDisplay(event) || eventFormatter.attributedString(from: event, with: roomState, error: formatterError) == nil {
```

Comment:
I wonder if we should test the formatterError as well here 🤔

Reply:
You're right. It's even safer :)